docs/tables/aws_pricing_product.md

```diff
@@ -0,0 +1,55 @@
+# Table: aws_pricing_product
+
+Returns costs, and offer definition, for the corresponding service.
+
+## Examples
```

Comment:
@aminvielledebatAtBedrock In the PR body (or in a new comment in it), can you please post the results (or a screenshot if it's too long), so we can see what some rows look like?

Reply:
Done in the PR description.

Feel free to ask me any other examples